src/components/ContactPolicy/ContactPolicy.js

```diff
@@ -42,14 +43,17 @@ export const ContactPolicy = InjectAppServices(
       } else if (formSubmitted) {
         message.text = _('contact_policy.success_msg');
         message.type = 'success';
+        message.delay = 3000;
       } else {
         return null;
       }
 
       return (
-        <li className="field-item">
-          <IconMessage {...message} className="bounceIn" />
-        </li>
+        <Flash delay={message.delay}>
```

Comment:
How about renaming this component into something like this?:
`ShowLikeFlash`
`FlashInAndOut`
`FlashAnimate`
Something that indicates that the component has the responsibility to show and hide at simple sight.

Reply:
Quizás pueda usarse un custom hook para ese componente. De tal forma que pueda ser customizado dónde se use. Algo así como lo hace https://www.npmjs.com/package/react-notifications-component

Reply:
ShowLikeFlash sounds good